ROADMAP.md

```diff
@@ -0,0 +1,13 @@
+# MDS-Core Roadmap
+
+Work-in-progress.  These are Lacuna priorities; feedback from other MDS-Core contributors is welcome.
+
+* Improved documentation for MDS-Core operation
+* Multi-MDS-version support (MDS 1.0 and later)
```

Comment:
Thanks for sharing! After our meeting we discussed this internally and wondered, if you plan this two-way, or if you are only thinking about the agency api?

Here is an example: Would GET /policies with an `Accept` header for Version `1.0` actually return the policy in `1.0`, even if the mds-core repo has already adapted MDS `1.2` (assuming there are any differences in a policy `1.0` vs. `1.2`)?

Reply:
I believe that's the case, yes.